src/useful-links.md

```diff
@@ -1,80 +1,55 @@
-# SORA Ecosystem
+# Useful Links
+Here are some links to articles covering the SORA philosphy, as well
+as technical resources explaining Polkadot and Substrate. 
+This section will be constantly updated. 
 
-https://wiki.sora.org/master
+## SORA Ecosystem
 
-https://medium.com/sora-xor/sora-the-new-economic-order-3ec3f0327e5a
+- An explanation on the [SORA Economic System](https://medium.com/sora-xor/sora-the-new-economic-order-3ec3f0327e5a)
 
-https://medium.com/sora-xor/many-worlds-one-economy-1ce709d4fb42
+- The SORA vision of [Many Worlds, One Economy](https://medium.com/sora-xor/many-worlds-one-economy-1ce709d4fb42)
 
-https://www.forbes.com/sites/tatianakoffman/2021/11/23/the-rise-of-decentralized-money-on-polkadotnew-algorithmic-stablecoin-launches-on-sora/?sh=722c2e6f31bc
+- A Forbes article covering [the launch of XSTUSD](https://www.forbes.com/sites/tatianakoffman/2021/11/23/the-rise-of-decentralized-money-on-polkadotnew-algorithmic-stablecoin-launches-on-sora/?sh=722c2e6f31bc)
 
-https://wiki.sora.org/polkaswap/what-is-polkaswap
+- A Forbes article covering [the launch of Polkaswap](https://www.forbes.com/sites/tatianakoffman/2021/04/27/the-rise-of-decentralized-exchanges-on-polkadot/?sh=39acc5e58169)
 
-https://www.forbes.com/sites/tatianakoffman/2021/04/27/the-rise-of-decentralized-exchanges-on-polkadot/?sh=39acc5e58169
 
-https://polkaswap.io/#/swap
+## Polkadot
 
-https://wiki.sora.org/sora-academy/accounts-in-sora
+- An explainer on [What is Polkadot?](https://wiki.polkadot.network/docs/getting-started)
 
-https://drive.google.com/file/d/1FWxZJs_a-ZghmSrgI31t7hEyLL8QUH2H/view?usp=sharing
+- An explainer on [what are Parachains](https://wiki.polkadot.network/docs/learn-parachains)
 
-https://medium.com/sora-article-translations
+### Polkadot Client
+- [polkadot.js documentation](https://polkadot.js.org/docs/)
 
-https://www.notion.so/Engineering-Wiki-a84d681ac67c44c5a924c786a34bd246
+- [Polkadot.js mirroring new types from Rust to
+JS](https://polkadot.js.org/docs/api/start/types.extend) and [polkadot.js mirroring new rpc’s from Rust to JS](https://polkadot.js.org/docs/api/start/rpc.custom)
 
-# Polkadot
+- Running [Polkadot Apps locally](https://github.com/polkadot-js/apps)
 
-What is Polkadot ?: https://wiki.polkadot.network/docs/getting-started
+- The [Rust cli client](https://github.com/paritytech/substrate-subxt)
 
-Parachains: https://wiki.polkadot.network/docs/learn-parachains
+- The [JS cli client](https://github.com/paritytech/substrate-cli-tools)
 
-# Substrate
+- [Decode Substrate](https://github.com/paritytech/desub)
 
-Getting started
-On Windows - https://substrate.dev/docs/en/knowledgebase/getting-started/windows-users
+## Substrate
 
-Unix based systems - https://substrate.dev/docs/en/knowledgebase/getting-started/#manual-installation
+- Some links to getting started using Substrate; 
```

Comment:
it can be just
```suggestion
Some links to getting started using Substrate; 
```
or a subsection like this

```suggestion
### Get started with Substrate 
```

+ whatever you choose, the same applies to the general knowledge line below